mixins/_visibility.pcss

```diff
@@ -22,6 +22,7 @@
 	padding: 0;
 	position: absolute;
 	width: 1px;
+	min-height: auto;
```

Comment:
just out of curiosity, was there a specific case where this is needed? usually, when using a utility such as this, we would want this to be the first thing we apply, so we wouldn't have a min-height set.

Reply:
This was needed for the Administration to hide some of the `input` fields, if you have a min height it still shows on the page the element and breaks certain pieces of the navigation.